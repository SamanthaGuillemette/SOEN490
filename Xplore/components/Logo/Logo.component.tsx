import { StyleSheet, View, Image } from "react-native";

interface LogoProps {}

export const Logo = ({}: LogoProps) => {
  return (
    <View style={styles.logo_container}>
      <Image style={styles.logo} source={require("../../assets/logo.png")} />
    </View>
  );
};

const styles = StyleSheet.create({
  logo: {
    width: 100,
    height: 100,
  },
  logo_container: {
    marginTop: 65,
    marginBottom: 5,
    borderRadius: 8,
    backgroundColor: "white",
    shadowColor: "#463FB0",
    shadowOpacity: 0.1,
    shadowOffset: {
      width: 0,
      height: 0,
    },
    elevation: 5,
  },
});
