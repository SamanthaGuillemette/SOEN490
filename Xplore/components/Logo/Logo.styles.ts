import { StyleSheet } from "react-native";

export default StyleSheet.create({
    logo: {
        width: 100,
        height: 100,
      },
      logo_container: {
        marginTop: 65,
        marginBottom: 5,
        borderRadius: 8,
        backgroundColor: "white",
        shadowColor: "#463FB0",
        shadowOpacity: 0.1,
        shadowOffset: {
          width: 0,
          height: 0,
        },
        elevation: 5,
      },
});
