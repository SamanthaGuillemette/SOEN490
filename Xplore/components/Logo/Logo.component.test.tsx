import React from "react";
import { cleanup } from "react-native-testing-library";
import { Logo } from "./Logo.component";
import renderer from "react-test-renderer";
import { useColorScheme } from "react-native";

afterAll(() => {
  cleanup();
});

describe("Logo should render correctly", () => {
  it("should render correctly", () => {
    const logo = renderer.create(<Logo />).toJSON;
    expect(logo).toMatchSnapshot();
  });
});
