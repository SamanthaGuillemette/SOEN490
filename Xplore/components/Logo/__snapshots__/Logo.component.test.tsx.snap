// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should render correctly should render correctly 1`] = `[Function]`;
